recipe/recipe.yaml

```diff
@@ -1,5 +1,6 @@
 context:
   version: 1.4.2
+  python_min: "3.9"
```

Comment:
However it looks like `rattler-build` doesn't pick this up. So it is being respecified in `context` here, which shouldn't be needed

Have raised in upstream issue: https://github.com/prefix-dev/rattler-build/issues/1248